src/theme/headerLowerNav.ts

```diff
@@ -14,17 +14,19 @@ const HeaderLowerNav = {
   baseStyle: {
     ul: {
       alignItems: "center",
-      fontSize: "1",
-      marginBottom: "xxs",
+      marginBottom: "0",
       marginLeft: "auto",
       whiteSpace: "nowrap",
     },
     li: {
-      letterSpacing: ".02em",
-      marginLeft: "xs",
+      marginEnd: { lg: "s", xl: "m" },
+      _last: {
+        marginRight: "0",
+      },
     },
     "li > a": {
       color: "ui.black",
+      fontSize: "desktop.subtitle.subtitle2",
```

Comment:
This comment didn't make it into my review.  Maybe I didn't hit the submit button?  Hmm, not sure.

This should be `subtitle1` for the largest breakpoint.